packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -681,7 +681,7 @@ - (void)invalidateLayer
   } else {
     if (!_borderLayer) {
       CALayer *borderLayer = [CALayer new];
-      borderLayer.zPosition = -1024.0f;
+      borderLayer.zPosition = 1024.0f;
```

Comment:
Hi, maybe you can check the code history, and it introduce borderLayer is because CSS draws the border behind the content :) .

https://github.com/facebook/react-native/pull/45973/files#diff-fce237eeca8733cbef40d994a0c0a154026be79e681b103b4afde567dd42330bL659-L661

Reply:
Yeah, css draws a border behind the component's children. In the provided solution, the border is still drawn behind the inner content of the element. I think that layer's `zPosition` applies only within a layer, so it shouldn't have an impact outside of that. The above problem looks like this after the changes on Fabric:

![image](https://github.com/user-attachments/assets/df45c174-a3ff-4b7b-b681-cff142d9abb9)

Reply:
Emm, `zPosition` also affects the `view` or `layer` for the same hierarchy, For example like below, setting border layer's `zPosition` to 1024 can cover the content of SubView1 I think. 

```
View
---border layer
---SubView1
------Subview of SubView1
```

Reply:
BTW, I'm trying to refactor the Text to support border, you can try #45972 and applied your code, you can see border is cover the text like below: :)

![image](https://github.com/user-attachments/assets/88044b5c-eead-40d1-9a0e-7fb902fd0b7e)

We hope to see this:
![image](https://github.com/user-attachments/assets/5181ccec-898b-4f71-abf5-a577d06d6aa4)

Reply:
Unfortunately, the iOS docs are not too descriptive about how the `zPosition` works 😕 I will try to think about other solution.